### DISCUSSION TITLE
"UPTANE" Capitalization

### DISCUSSION THREAD

**Discussion by PhilPNguy** [uptane/uptane.github.io#151] [Ideas]
I don't think we should have "UPTANE" all capitalized in the front page since everywhere else it is either the logo or as "Uptane"
We should decide one of the following:

1. "Uptane" normal text
2. Use the Uptane logo

**Comment by jhdalek55** [comment]
Good observation. I would suggest changing the spelling of Uptane at top with the initial capital letter only for consistency.

**Reply by hexsecs** [reply]
We have the logo in a couple of places in the homepage so maybe just the camel case version "Uptane".

**Comment by hexsecs** [comment]
We resolved this in #168
